examples/knowledge_tuning/instructlab/document_collection/ibm-annual-report/qna.yaml

```
@@ -0,0 +1,100 @@
+domain: Standard Procedure Mannual
+seed_examples:
+- context: |
+    ## Management Discussion
+
+    International Business Machines Corporation and Subsidiary Companies
+
+    Software revenue of $27,085 million increased 8.3 percent as reported (9.0 percent adjusted for currency) in 2024 compared to the prior year, reflecting growth across all lines of business with double-digit growth in Red Hat and Automation and high single-digit growth in Transaction Processing. This revenue performance reflects the investments we have been making in Software, both organically and through acquisitions. Our investments in generative AI are contributing to growth, as we had strong demand for our generative AI products such as watsonx, Concert and our AI assistants. We also launched new products in the fourth quarter of 2024 including our next generation of watsonx Code Assistant and Guardium Quantum Safe. In 2024, we also had increased revenue contribution from acquisitions compared to the prior year, including Apptio and StreamSets and webMethods. In addition, we had solid growth in our recurring revenue and double-digit growth in our transactional software revenue in 2024, as we accelerated growth through innovation across our Software portfolio.
+
+    Hybrid Platform & Solutions revenue of $18,808 million increased 8.1 percent as reported (8.7 percent adjusted for currency) in 2024 compared to the prior year. Within Hybrid Platform & Solutions, Red Hat revenue increased 11.4 percent as reported (12.0 percent adjusted for currency), which reflects the continued demand for our hybrid cloud solutions as clients are prioritizing application modernization on OpenShift containers and Ansible automation to optimize their IT spending and reduce operational complexity. In 2024, we had double-digit revenue growth in OpenShift and Ansible, and high single-digit revenue growth in RHEL. The growth in OpenShift revenue reflects increased volume in OpenShift Virtualization engagements, and we exited 2024 with OpenShift annual recurring revenue of $1.4 billion. Automation revenue increased 14.2 percent as reported (14.8 percent adjusted for currency), driven by our SaaS subscription offerings such as AIOps and Management, which includes the higher revenue contribution from Apptio. Data & AI revenue increased 1.6 percent as reported (2.2 percent adjusted for currency), with strong growth in Data Fabric and our AI assistant for Customer Care, driven by client demand for our watsonx platform offerings, and strength in asset and supply chain management software which helps clients run sustainable operations. Security revenue increased 0.8 percent as reported (1.5 percent adjusted for currency), with revenue growth in data security and identity and access management, partially offset by a revenue decline in security threat management.
+  questions_and_answers:
+    - question: What contributed to IBM's 8.3% increase in Software revenue in 2024?
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)